locksmith/__tests__/controllers/metadataController/tokenData.test.ts

```diff
@@ -120,11 +126,11 @@ describe('Requesting Token Data', () => {
       },
     })
 
-    mockWeb3Service.isLockManager = jest.fn(() => Promise.resolve(false))
+    mockWeb3Service.isLockManager = vi.fn(() => Promise.resolve(false))
   })
 
   describe("when persisted data doesn't exist", () => {
-    it('returns wellformed data for Week in Ethereum News', async () => {
+    it.skip('returns wellformed data for Week in Ethereum News', async () => {
```

Comment:
is that normal for these to be skipped?

Reply:
Yes. 3 are skipped because they need to change. I will do it after this PR.

Reply:
Could do these changes _before_ this PR so we don't forget?